### ISSUE TITLE
jupyter nbconvert --execute requires --allow-errors

### ISSUE THREAD

**Issue by molpopgen** [google/evcxr#153]
Thanks for this great resource!

I'm working on writing some code to be executed as part of a book.  I'm running into an issue during standalone execution.

Take the following small notebook:

```
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "forced-window",
   "metadata": {},
   "outputs": [],
   "source": [
    "println!(\"Howdy\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Rust",
   "language": "rust",
   "name": "rust"
  },
  "language_info": {
   "codemirror_mode": "rust",
   "file_extension": ".rs",
   "mimetype": "text/rust",
   "name": "Rust",
   "pygment_lexer": "rust",
   "version": ""
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
```

Attempting to convert via `jupyter nbconvert --execute --to html test.ipynb` gives:

```
[NbConvertApp] Converting notebook test.ipynb to html
[NbConvertApp] Executing notebook with kernel: rust
[NbConvertApp] ERROR | Error while converting 'test.ipynb'
Traceback (most recent call last):
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/nbconvertapp.py", line 410, in export_single_notebook
    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/exporter.py", line 179, in from_filename
    return self.from_file(f, resources=resources, **kw)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/exporter.py", line 197, in from_file
    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/html.py", line 95, in from_notebook_node
    return super(HTMLExporter, self).from_notebook_node(nb, resources, **kw)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/templateexporter.py", line 307, in from_notebook_node
    nb_copy, resources = super(TemplateExporter, self).from_notebook_node(nb, resources, **kw)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/exporter.py", line 139, in from_notebook_node
    nb_copy, resources = self._preprocess(nb_copy, resources)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/exporters/exporter.py", line 316, in _preprocess
    nbc, resc = preprocessor(nbc, resc)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/preprocessors/base.py", line 47, in __call__
    return self.preprocess(nb, resources)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/home/kevin/.local/lib/python3.8/site-packages/nbconvert/preprocessors/execute.py", line 450, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, reply['content'])
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
println!("Howdy")
------------------


<Error>: 
```

However, `jupyter nbconvert --execute --allow-errors --to html test.ipynb` works and gives:

```
[NbConvertApp] Converting notebook test.ipynb to html
[NbConvertApp] Executing notebook with kernel: rust
[NbConvertApp] Writing 272666 bytes to test.html
```

Ideally, I'd like to avoid suppressing all errors when working on the book.  I'm writing it as a [jupyter book](https://www.jupyterbook.org), and need the following front matter to mimic the working output from above:

```yaml
---
jupytext:
  formats: md:myst
  text_representation:
    extension: .md
    format_name: myst
kernelspec:
  display_name: rust
  language: rust
  name: rust
execution:
  allow_errors: true
---
```

**Comment by davidlattimore** [comment]
Thanks the report. I'd never tried nbconvert before. I've confirmed that I can reproduce the problem. I'll try to take a look in the next few days. Hopefully should be easy to fix once I figure out what request to the kernel is returning an error.

**Comment by davidlattimore** [comment]
Well, that turned out to be a silly mistake on my part. I was sending an error status when OK and an OK status when there was an error. Oops!

**Comment by molpopgen** [comment]
Thanks.  I cargo installed the new version and it works.  Any idea re: a point release to get the bug fix on crates.io?

**Comment by davidlattimore** [comment]
I've just released 0.8.1 which includes this fix
